ck.sh

```diff
@@ -96,16 +95,17 @@ case "$COMMAND" in
 						--no-ie-checks 
 				fi
 
-				# Remove custom skin from ckeditor-dev
-				rm -rf skins/moono-lexicon
-
 				# Remove old build files.
 				rm -rf ../ckeditor/*
 
 				# Replace with new build files.
 				cp -r dev/builder/release/ckeditor/* ../ckeditor/
 
 				cd ..
+
+				# Remove temporal directory
+				rm -rf temp
```

Comment:
In case we don't get to this line, might want to add `/temp` to `.gitignore` file so that it doesn't clutter `git status` output.

Reply:
Sure